guides/common/assembly_monitoring-project-resources.adoc

```diff
@@ -4,15 +4,15 @@ include::modules/con_monitoring-project-resources.adoc[]
 
 include::modules/ref_using-the-project-content-dashboard.adoc[leveloffset=+1]
 
-include::modules/proc_managing-tasks.adoc[leveloffset=+2]
+include::modules/ref_managing-tasks.adoc[leveloffset=+2]
```

Comment:
```suggestion
include::modules/ref_managing-tasks.adoc[leveloffset=+1]
```

out of scope: looking at the built docs, it's not clear to me my this is one level lower: https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#Managing_Tasks_admin as far as I can tell, it's not directly related to the module above.